apps/frontend/src/pages/Homepage.tsx

```diff
@@ -170,6 +181,57 @@ const Homepage = () => {
 
   return (
     <>
+      {charityEvent != "" && (
+        <div style={{ position: "relative", height: "80px" }}>
```

Comment:
We should make this clickable to the charities page, I'll be doing something similar on every listing
```suggestion
        <div style={{ position: "relative", height: "80px", "&:hover":  {cursor: "pointer"}}} onClick={()=>{navigate("/charities")}} >
```